src/theory/partition_generator.cpp

```diff
@@ -236,32 +352,23 @@ Node PartitionGenerator::makeFullTrailPartitions(LiteralListType litType,
     // total number of cubes/rows
     size_t total = pow(2, numVar);
 
-    // resultNodeLists is built column by column. 
+    // resultNodeLists is built column by column.
     std::vector<std::vector<Node> > resultNodeLists(total);
 
     // t is used to determine whether to push the node or its not_node.
     bool t = false;
 
-    // numConsecutiveTF tracks how many times the node should be consectuively 
-    // true or false in a column.
-    // For example, if numVar=3:
-    // x y z
-    // T T T
-    // T T F
-    // T F T
-    // T F F
-    // F T T
-    // F T F
-    // F F T
-    // F F F
-    // For the first column, numConsecutiveTF = 4, then 2 for the second column, 
-    // and 1 for the third column.
+    // numConsecutiveTF tracks how many times the node should be
+    // consectuively true or false in a column. For example, if numVar=3: x
```

Comment:
Oops, this long comment got collapsed into nonsense with the auto-formatting. Will fix it.